scico/linop/_xray.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2020-2023 by SCICO Developers
+# All rights reserved. BSD 3-clause License.
+# This file is part of the SCICO package. Details of the copyright and
+# user license can be found in the 'LICENSE' file distributed with the
+# package.
+
+
+"""
+X-ray projector classes.
+"""
+from functools import partial
+from typing import Optional
+
+import numpy as np
+
+import jax
+import jax.numpy as jnp
+from jax.typing import ArrayLike
+
+from scico.typing import Shape
+
+from ._linop import LinearOperator
+
+
+class XRayProject(LinearOperator):
```

Comment:
Naming is not consistent with existing CT projector classes: worth discussion.

Reply:
discussion: call all of them radon transform, clarify in the docs

Reply:
New discussion: `XRayTransform`